Packs/CommonTypes/IndicatorFields/indicatorfield-positivedetections.json

```diff
@@ -18,7 +18,7 @@
     "breachScript": "",
     "editForm": true,
     "description": "Number of engines that positively detected the indicator as malicious",
-    "associatedToAll": true,
+    "associatedToAll": false,
     "associatedTypes": null,
```

Comment:
Dont you think we should keep that field? (I know in the layout we have a section to see the verdict for each integration) but a field that we can use in playbooks and dashboards?

Reply:
Leave it where? it should be an integration specific field (like in VT) what else do we use it for?

Reply:
ok agree :)